src/components/structured-text-block/structured-text-block.vue

```diff
@@ -249,13 +262,66 @@
   }
 
   .structured-text__blue-text--center .structured-text__buttons-list {
-    align-self: center;
+    margin-left: auto;
+    margin-right: auto;
   }
 
   .structured-text__image-with-caption {
     margin: var(--spacing-big) 0;
   }
 
+
+  @media (max-width: 719px) {
+    .structured-text__image-with-caption--left + .structured-text__image-with-caption--right {
+      margin-top: var(--spacing-small)
+    }
+  }
+
+  @media (min-width: 720px) {
+    .structured-text__image-with-caption--narrow {
+      margin-left: auto;
+      margin-right: auto;
+      max-width: 80%
+    }
+
+    .structured-text__image-with-caption--left,
+    .structured-text__image-with-caption--right {
+      width: 50%;
+    }
+
+    .structured-text__image-with-caption--left {
+      padding-right: var(--spacing-small);
+      float: left;
+    }
+
+    .structured-text__image-with-caption--right {
+      padding-left: var(--spacing-small);
+      float: right;
+    }
+
+    .structured-text__image-with-caption--left:has(+ *:not(.structured-text__image-with-caption--right)) {
+      margin-top: 0;
+    }
+
+    /* When combining the selector with the :has selecotor, this selector doesn't work anymore in unsupported browsers  */
```

Comment:
```suggestion
    /* When combining the selector with the :has selector, this selector doesn't work anymore in unsupported browsers  */
```

Reply:
- I don't understand this comment, is this a fallback of the selector above?
- Firefox doesn't support `:has` at all right?
- I find all of these selectors hard to read, wonder if there's an way to simplify the CSS by perhaps adding some more logic to the template?

Reply:
- No this is not a fallback. But when grouping a supported selector with an unsupported selector, both won't get applied.
- Firefox does not support has indeed. So in firefox you get a bit of a weird margin
- The issue is that's it contextual base, based on the previous element. In the `renderBlock` function we unfortunately don't have access to the previous element. Or we would have to dig really deep into the structured text ast tree. For now I've added an explanation of what the selector selects. Is that workable for you?